demophin.json

```diff
@@ -7,10 +7,18 @@
                 "path": "./erg.dat",
                 "description": "The English Resource Grammar"
             }
+            , {
+                "name": "JACY",
+                "path": "./jacy.dat",
+                "description": "The Jacy Japanese Grammar",
+                "preprocessor" : "pyjacy",
+                "aceopts" : [ "-y" ]
```

Comment:
I like the idea of a preprocessor specification here. I'm not certain I like the idea that it's a Python module.

Also, `aceopts` should just be `cmdargs` as it is below, otherwise it seems like something different. We might also consider generalizing ACE overrides here with an `ace` subkey:

```json
...
    {
        "name": "Jacy",
        ...,
        "ace": {
            "cmdargs": ["-n", "10"]
        }
    }
...
```

That way we can override other things, like the executable version, etc. But maybe that makes things too complicated? And would `preprocessor` go under `ace` here, or outside of it?